upcloud/resource_upcloud_server.go

```diff
@@ -400,7 +409,11 @@ func resourceUpCloudServerRead(ctx context.Context, d *schema.ResourceData, meta
 		return diag.FromErr(err)
 	}
 	_ = d.Set("hostname", server.Hostname)
-	_ = d.Set("title", server.Title)
+	if server.Title != cloudServerDefaultTitleFromHostname(server.Hostname) {
+		_ = d.Set("title", server.Title)
+	} else {
+		_ = d.Set("title", nil)
```

Comment:
What's the effect of setting this to nil here? Will TF make sure it's the default one?

Reply:
This sets the `title` value in `tfstate` to empty string. And I'm not sure if this is actually the desired behaviour. `title` field is marked as computed, so if I don't provide one, I would expect the computed value to end up in my tfstate.